manager/controllers/package_detail_test.go

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import (
+	"app/base/core"
+	"app/base/utils"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestLatestPackage(t *testing.T) {
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", "/packages/kernel", nil)
+	core.InitRouterWithParams(PackageRouter, "3", "GET", "/packages/:package_param").
+		ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+	var output PackageDetailResponse
+	assert.Greater(t, len(w.Body.Bytes()), 0)
+	ParseReponseBody(t, w.Body.Bytes(), &output)
+	assert.Equal(t, "kernel", output.Name)
+	assert.Equal(t, "RH-7", output.Data.Advisory.Name)
+	assert.Equal(t, "5.6.13-201.fc31.x86_64", output.Data.Package.EVRA)
+}
+
+func TestEvraPackage(t *testing.T) {
```

Comment:
Please add also some edge cases (non-existing package_name, non-existing nevra, existing package_name with non-existing version). Please ensure to return correct status codes. See `AdvisoryDetailHandler` and use similar way.